### PR TITLE
Refs task 7509 Create Package

### PR DESCRIPTION
**Pull request by ngochieu1297**
## [Pull request title]
Create Package
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7509

### CODE REVIEW

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/java/com/example/music_58/data/repository/TrackDataRepository.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.example.music_58.data.repository;
+
+public class TrackDataRepository {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/data/repository/TrackDataRepository.java]:
```suggestion
public class TrackRepository {
```

File: app/src/main/java/com/example/music_58/util/StringUtil.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.example.music_58.util;
+
+public class StringUtil {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/util/StringUtil.java]:
```suggestion
public class StringUtils {
```

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,8 +1,22 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".ui.MainActivity">
+
+    <SearchView
+        android:layout_width="0dp"
+        android:layout_height="50dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main.xml]: hardcode

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,8 +1,22 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hot today"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main.xml]: 18 19 20hardcode

**Comment by ngochieu1297** [comment]
ready

**Pull request merged**